egg/core/continous_communication.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Callable
+
+import torch.nn as nn
+import torch.nn.functional as F
+
+from egg.core import Interaction
+
+
+class ContinuousLinearSender(nn.Module):
+    def __init__(
+        self,
+        agent: nn.Module,
+        encoder_input_size: int,
+        encoder_hidden_size: int = 64,
+        num_layers: int = 1,
+        activation: str = "relu"
+    ):
+        super(ContinuousLinearSender, self).__init__()
+
+        activations = {"relu": F.relu, "tanh": F.tanh, "leaky_rely": F.leaky_relu, "identity": nn.Identity()}
+        try:
+            self.activation = activations[activation.lower()]
+        except KeyError:
```

Comment:
let it crush? otherwise we'd type `rely` instead of `relu` and it will silently do different things